Header header
#
# map resolution in meters 
float32 cell_size 
#
# origin_row_index
uint32 origin_row_index
#
# origin_col_index
uint32 origin_col_index 
# 
# msg defines the properties of any 2D Map